$NetBSD: patch-default.conf,v 1.3 2021/03/14 08:44:01 nia Exp $

* Use more useful example URLs
* Add variable for archive file extension

--- default.conf.orig	2013-07-28 21:04:02.000000000 +0000
+++ default.conf
@@ -7,9 +7,13 @@
 # configuration variables are listed in this sample configuration file.
 
 # Path to the release files (local path or remote URL).
-#RELEASEDIR="ftp://ftp.NetBSD.org/pub/NetBSD/NetBSD-6.0/$(uname -m)"
 #RELEASEDIR="ssh://me@builder.example.net/home/sysbuild/release/$(uname -m)"
-RELEASEDIR="/home/sysbuild/release/$(uname -m)"
+#RELEASEDIR="/home/sysbuild/release/$(uname -m)"
+#RELEASEDIR="https://cdn.NetBSD.org/pub/NetBSD/NetBSD-9.1/$(uname -m)"
+# Upgrade to the latest snapshot of the current branch
+#RELEASEDIR="https://nycdn.netbsd.org/pub/NetBSD-daily/HEAD/latest/$(uname -m)"
+# Upgrade to the latest snapshot of a stable branch
+RELEASEDIR="https://nycdn.netbsd.org/pub/NetBSD-daily/netbsd-9/latest/$(uname -m)"
 
 # Name of the kernel to be installed.
 KERNEL=AUTO  # Guess from /netbsd (requires config(1)).
@@ -34,3 +38,7 @@ POSTINSTALL_AUTOFIX="obsolete"
 # do "sysupgrade etcupdate" using the same etc.tgz file downloaded during
 # the upgrade, you will need the same etc.tgz file to be present.
 #AUTOCLEAN=no
+
+# Archive file extension of the sets. For now it can be either 'tgz' or
+# 'tar.xz'.
+#ARCHIVE_EXTENSION=tar.xz
